core_lib/helpers/func_utils.py

```diff
@@ -46,7 +46,7 @@ def get_func_parameters_as_dict(func, *args, **kwargs) -> dict:
         elif param in parameters and parameters[param].default != inspect.Parameter.empty:

```

Comment:
question please. 
how dose this effect `build_value_by_func_parameters` and it's tests